app/appconfig.go

```diff
@@ -7,11 +7,16 @@ import (
 )
 
 type appConfig struct {
-	ListenAddr  string
-	Verbose     bool
-	JSON        bool
-	LogRequests bool
-	APIOnly     bool
+	ListenAddr    string
+	TLSListenAddr string
+	TLSCertFile   string
```

Comment:
nit: put the `TLS` fields in their own "section" (just separated by whitespace) like `Jaeger`, `Tracing`, `DBMax` etc..